noxfile.py

```diff
@@ -16,13 +16,6 @@
     "SUNDIALS_INST": f"{homedir}/.local",
     "LD_LIBRARY_PATH": f"{homedir}/.local/lib:",
 }
-# Do not stdout ANSI colours on GitHub Actions
-if os.getenv("CI") == "true":
-    os.environ["NO_COLOR"] = "1"
-    # The setup-python action installs and caches dependencies by default, so we skip
```

Comment:
Not needed anymore after we use separate jobs instead of running tests sequentially (#3264). I added coloured outputs to GitHub Actions too